roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml

```diff
@@ -1,4 +1,22 @@
 ---
+# Disable external communication for {{ _cluster_component }}
+- name: Disable external communication for logging-{{ _cluster_component }}
+  oc_service:
+    state: present
+    name: "logging-{{ _cluster_component }}"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    selector:
+      component: "{{ _cluster_component }}"
+      provider: openshift
+      connection: blocked
```

Comment:
I presume this simply selects no pods since the pods dont have this label?

Reply:
correct